test/core/security/security_connector_test.cc

```diff
@@ -161,66 +161,51 @@ static int check_identity(const grpc_auth_context* ctx,
   return 1;
 }
 
-static int check_x509_cn(const grpc_auth_context* ctx,
-                         const char* expected_cn) {
-  grpc_auth_property_iterator it = grpc_auth_context_find_properties_by_name(
-      ctx, GRPC_X509_CN_PROPERTY_NAME);
+static int check_property_with_single_value(
```

Comment:
nit: I think the name `with_single_value` doesn't bring any more information, as there is no function called `check_property_with_multiple_values`, etc.
Maybe we can just call it `check_property` or `check_property_by_name`.